livekit-agents/livekit/agents/pipeline/io.py

```diff
@@ -141,6 +141,20 @@ def flush(self) -> None:
         ...
 
 
+class MultiTextSink(TextSink):
+    def __init__(self, sinks: list[TextSink]) -> None:
+        self._sinks = sinks
+
+    async def capture_text(self, text: str, *, segment_id: str | None = None) -> None:
+        await asyncio.gather(
```

Comment:
Let's keep the public API minimal for V1.0, I wouldn't expose it for now and keep this class private.